Update index.md CSV Download Accounts update (#5618)
segmentio/segment-docs: src/engage/audiences/index.md

> info ""
> Generating a CSV can take a substantial amount of time for large audiences. After you generate the CSV file, leave the modal window open while Segment creates the file. (If the audience recalculates between when you click Generate and when you download the file, you might want to regenerate the file. The CSV is a snapshot from when you clicked Generate, and could be outdated.)

> > note ""

Reviewer: ```suggestion

> warning ""
```

> > Account traits and Account identifiers are not possible to add to the CSV download with Account Level audiences, because every row listed is a user, and since account traits and identifiers only exist on accounts, then they would not technically exist as a user's custom trait and appear on the CSV.

Reviewer: ```suggestion
> You can't add account traits and identifiers using the CSV downloader with account level audiences. This is because every row listed in the CSV file is a user, and since account traits and identifiers only exist on accounts, they wouldn't exist as a user's custom trait and appear on the CSV.
```
